packages/busybox.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Busybox < Package
+  description 'BusyBox is a software suite that provides several Unix utilities in a single executable file'
+  homepage 'https://busybox.net'
+  compatibility 'all'
+  version '1.33.0'
+  source_url 'https://www.busybox.net/downloads/busybox-1.33.0.tar.bz2'
+  source_sha256 'd568681c91a85edc6710770cebc1e80e042ad74d305b5c2e6d57a5f3de3b8fbd'
+
+  def self.patch
+      # There's probably a way to use ruby to download this file. If anyone who knows ruby wants to change it, please do so.
+      # When the package is merged, change the curl command to:
+      #system "curl -#L https://raw.github.com/skycocker/chromebrew/master/misc/busybox-config -o ./.config"
+      system "curl -#L https://raw.github.com/saltedcoffii/chromebrew/patch-16/misc/busybox-config -o ./.config" 
```

Comment:
Would a sha256sum check make sense here?  Probably should put this on a `busybox-config` branch instead of `patch-16` which doesn't make sense to a casual observer.

Reply:
Remove `misc/busybox-config` since we're downloading here?

Reply:
I guess i don't know where you want the busybox config file and in what branch in which repository (mine or the main chromebrew one). Let me know and i can make those changes/pulls, i just don't know what is wanted.

Reply:
It probably makes sense to add the config in `/user/chronos/home` or `~`.  Maybe `#{CREW_DEST_HOME}/.config/busybox-config`?  This would require `depends_on 'xdg_base'`.